src/backends/tensorflow/base_backend.cc

```diff
@@ -649,6 +687,26 @@ BaseBackend::Context::Run(
       skip_element_cnt = false;
     }
 
+    const DimsList& output_dims = (output_config->has_reshape())
```

Comment:
So can we not validate the shape at init time? For saved-model I know we can, what about graphdef?

Reply:
I tested it, it works for graphdef. The QA models include some graphdef models as well and it runs successfully with them.

Reply:
I was asking if we can test the graphdef tensor shapes against the model config at init time instead of runtime (like your change here). Since we already can check saved-model at init time it would be good if we could also check graphdef at init time, then we wouldn't need this runtime check.

Reply:
Got it. Will try to do this at init time.

Reply:
Not possible for all graphdef models.

Reply:
When possible the model's tensor shape and datatype are used for validation of GraphDef models in Init. If not, the same is done during runtime.